library-client-rtsp/src/main/java/com/alexvas/rtsp/widget/RtspSurfaceView.kt

```diff
@@ -30,12 +30,12 @@ open class RtspSurfaceView: SurfaceView {
     private var requestVideo = true
     private var requestAudio = true
     private var rtspThread: RtspThread? = null
-    private var videoFrameQueue = FrameQueue(60)
+    var videoFrameQueue = FrameQueue(200)
     private var audioFrameQueue = FrameQueue(10)
     private var videoDecodeThread: VideoDecodeThread? = null
     private var audioDecodeThread: AudioDecodeThread? = null
-    private var surfaceWidth = 1920
-    private var surfaceHeight = 1080
+    private var surfaceWidth = 720
```

Comment:
What is this?

Reply:
We did that according to our application needs. But now working on it to change it dynamically.